### PR TITLE
JSession: Removed duplicated code, code style

### PR DESCRIPTION
**Pull request by piotr-cz**
Minor fixes not affecting functionality

### CODE REVIEW

File: libraries/joomla/session/session.php [hunk 1]
```diff
@@ -408,11 +402,7 @@ public function isActive()
 	public function isNew()
 	{
 		$counter = $this->get('session.counter');
-		if ($counter === 1)
-		{
-			return true;
-		}
-		return false;
+		return (bool) ($counter === 1)
```
Comment (realityking) [inline libraries/joomla/session/session.php]: You're missing the ; at the end of the line.

Comment (piotr-cz) [inline libraries/joomla/session/session.php]: Thanks, fixed.

I don't understand why there is a typecast to boolean in `JSession->isActive` and I repeated the style. Should I remove these?

Comment (realityking) [inline libraries/joomla/session/session.php]: However you want, it's not needed but personally I don't mind it.

Comment (piotr-cz) [inline libraries/joomla/session/session.php]: Lets keep it then

**Pull request merged**